tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -37,23 +37,36 @@ TiDB Cloud provides some connection examples in the **Connect** dialog. You can
 
 Generally, enabling TLS and offering a CA root path to authenticate the server is a good practice to prevent a man-in-the-middle attack. Different clients have different operations in the TLS connection. Enable TLS and verify the server according to your actual use of the client.
 
-The following examples show the connection string in MySQL CLI client, MyCLI client, Java, Python, Go, and Node.js:
+The following examples show the connection string in MySQL CLI client, MariaDB CLI client, MyCLI client, Java, Python, Go, and Node.js:
```

Comment:
```suggestion
The following examples show the connection string in MySQL Client, MariaDB Client, MyCLI client, Java, Python, Go, and Node.js:
```

Reply:
I deliberately changed to this since it's official called MySQL Command-Line Client. I think CLI could make it more clear. Search MySQL Client on Google will also lead to results like MySQL Shell.

Reply:
Changing it from "MySQL Client" to "MySQL CLI Client" to avoid getting "MySQL Shell" results from Google doesn't seem to work:

![image](https://user-images.githubusercontent.com/1272980/201066919-e3f369f1-7dfd-4b6a-89fb-69398f7303d9.png)

Looks like "MySQL Client" is the common way of naming this: https://trends.google.com/trends/explore?q=%22MySQL%20Client%22,%22MySQL%20CLI%20Client%22

Reply:
I would like to also change our connect tab to be consistent with this. So I follow PlanetScale and Cockroach Serverless:
![image](https://user-images.githubusercontent.com/3690895/201081250-a6504d49-0347-4b85-9e79-ad1e64d0bb0f.png)
![image](https://user-images.githubusercontent.com/3690895/201081281-ce69d86c-3730-4c74-8e0b-68d2d2d42278.png)
Both "Client" and "CLI" are okay to me.